config/runtime.go

```diff
@@ -26,10 +26,25 @@ import (
 type Runtime struct {
 	DiagnosticsConfig DiagnosticsConfig         `yaml:"diagnostics,omitempty"`
 	HealthChecks      HealthChecksConfig        `yaml:"health-checks,omitempty"`
+	AuditConfig       *AuditConfig              `yaml:"audit,omitempty"`
 	LoggerConfig      *LoggerConfig             `yaml:"logging,omitempty"`
 	ServiceDiscovery  httpclient.ServicesConfig `yaml:"service-discovery,omitempty"`
 }
 
+type AuditConfig struct {
+	Deployment string `yaml:"deployment,omitempty"`
+
+	Product string `yaml:"product-name,omitempty"`
```

Comment:
Is there a reason to not use the existing `ProductName` and `ProductVersion` from install config?

Reply:
Good question -- so this config mirrors `AuditConfiguration.java` on the Java side. @IlanaRadinsky do you have context for why this is defined separately? Is it because in Java it's hard to read off the "general" one?

Reply:
I don't think there's a good reason for this - it should work to just use the values from the install config.